templates/components/streamfield/blocks/quote_block.html

```diff
@@ -1,5 +1,19 @@
-{% verbatim %}
-<div class="">
-    {{ self }}
-</div>
-{% endverbatim %}
+<div class="rich-text mb-10 md:mb-20 last:mb-0">
+    <h2 class="font-serif4 text-3xl md:text-5xl font-semibold text-center
```

Comment:
Do we intentionally want the quote and attribution to be headings (in terms of semantics)? We can keep the large font with `text-3xl md:text-5xl`, but I don't know if it should be a heading.

Also, I'm not sure about `text-center`. I've heard opinions that it's less readable. Though it's probably not that big of an issue for brief text like quotes with a large font.

Reply:
I chose text center mostly because it helps the text stand out a bit more from the typical paragraphs and headings on the page. I think it helps make the quotes pop a bit without having to get too fancy design-wise.

I am honestly not sure whether a heading would work better semantically. I think it would probably be an odd screenreader experience to put block quotes at the same level as headings though for navigation purposes, so I went ahead and made these paragraph tags again.